### ISSUE TITLE
[Design Tokens] Contrast ratio on disabled elements

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#2999]
### Provide a general summary of the issue here

Right now contrast of `disabled` elements is mostly 3.45:1 for white background and it's not a WCAG violation, as disabled state doesn't need to have any contrast. But there are approaches that says even disabled state should have contrast of at least 4.5:1 with background, and at least 3:1 with element of that role in active state. 
This make it easier to read elements in disabled state and is generally more accessible. 
Also a11y automated tests would be more reliable, as there would be no exception for `color-contrast` rule.

Let's discuss if we can change color for `disabled` and maybe investigate on how big of a change this would be.

### 👤 Your name

SLDS team

### 🧢 Your Product/Team

SLDS team

**Comment by arecuenco-dsgn** [comment]
Let's wait for Danielle to discuss this
